sdcm/mgmt/common.py

```diff
@@ -1,13 +1,29 @@
+from enum import Enum
 import logging
 import datetime
+import yaml
 
-from enum import Enum
+from sdcm.utils.distro import Distro
 
 
 DEFAULT_TASK_TIMEOUT = 7200  # 2 hours
 LOGGER = logging.getLogger(__name__)
 
 
+def distro_to_name(distro_object):
```

Comment:
function name should be something like `verify_distro_supported` or something in that sense